### PR TITLE
Add base directory with initial docker file

### PR DESCRIPTION
**Pull request by MinsukJi-NOAA**

### CODE REVIEW

File: base/Dockerfile.ubuntu_20.04.gnu [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+           libmpich-dev \
+           libtool \
+           libexpat1-dev \
+           make \
+           pkg-config \
+           vim \
+           wget \
+           unzip \
+           python3-dev \
+           python3-pip
```
Comment (aerorahul) [inline base/Dockerfile.ubuntu_20.04.gnu:23]: we need tar, gzip, doxygen as well.

**Review by aerorahul — COMMENTED** [review]
looks good but is missing a few needed unix tools.

File: base/Dockerfile.ubuntu_20.04.gnu (hunk 1 above)
Comment (MinsukJi-NOAA) [reply base/Dockerfile.ubuntu_20.04.gnu:23]: tar and gzip are available without separately installing. doxygen takes up about 160MB of space; can that be optional if a user need it?

Comment (aerorahul) [reply base/Dockerfile.ubuntu_20.04.gnu:23]: I see. Ok. we can leave doxygen for now.

**Review by aerorahul — APPROVED** [review]
We should make a practice to keep a `README.md` updated with every PR, so that there is documentation.  For e.g. This is a base container with GNU and unix tools.

**Comment by MinsukJi-NOAA** [comment]
> We should make a practice to keep a `README.md` updated with every PR, so that there is documentation. For e.g. This is a base container with GNU and unix tools.

Yes, I agree.  We should have a `README.md` for each of, for example, `base`, `app`, and `data`

**Pull request merged**